python_modules/dagster/dagster/_core/definitions/decorators/schedule_decorator.py

```diff
@@ -260,6 +262,10 @@ def monthly_schedule(
         default_status (DefaultScheduleStatus): Whether the schedule starts as running or not. The default
             status can be overridden from Dagit or via the GraphQL API.
     """
+    job_name = check.not_none(
+        canonicalize_backcompat_args(job_name, "job_name", pipeline_name, "pipeline_name", "1.2.0")
+    )
```

Comment:
would advise stacking the removal PR for these now while its still fresh to prevent forgetting about it

Reply:
Yep I have a draft PR here that contains several of these 1.2 removals